src/ape/managers/networks.py

```diff
@@ -437,10 +438,28 @@ def get_ecosystem(self, ecosystem_name: str) -> EcosystemAPI:
             :class:`~ape.api.networks.EcosystemAPI`
         """
 
-        if ecosystem_name not in self.ecosystem_names:
-            raise EcosystemNotFoundError(ecosystem_name, options=self.ecosystem_names)
+        if ecosystem_name in self.ecosystem_names:
+            return self.ecosystems[ecosystem_name]
 
-        return self.ecosystems[ecosystem_name]
+        elif ecosystem_name in PUBLIC_CHAIN_META:
```

Comment:
```suggestion
        elif ecosystem_name.lower() in PUBLIC_CHAIN_META:
            ecosystem_name = ecosystem_name.lower()
```

The names in evmchains json use pascal case so this should flatten case if it falls back far enough to check evmchains so it will justwork(tm) for users

Reply:
although really it should slugify for chains like polygon-zkevm